components/jeune/FormulaireRechercheDossier.tsx

```diff
@@ -26,7 +26,9 @@ function FormulaireRechercheDossier({
 
   const validate = () => {
     if (numeroDossier === '') {
-      setMessageErreur('Veuillez renseigner un numéro de dossier')
+      setMessageErreur(
+        'Le champ est numéro de dossier est vide. Renseigner un numéro. exemple : 123456'
```

Comment:
```suggestion
        'Le champ "Numéro de dossier" est vide. Renseigner un numéro. exemple : 123456'
```
il y a une coquille dans le texte du ticket, à modifier :-)